x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/WaterfallItem.tsx

```diff
@@ -115,6 +119,9 @@ export function WaterfallItem({
   const left = (item.offset / totalDuration) * 100;
   const Label = item.docType === 'transaction' ? TransactionLabel : SpanLabel;
 
+  // Note: the <Prefix> appears *after* the item name in the DOM order
+  // because this label is styled with "direction: rtl;" so that the name
+  // itself doesn't flow outside the box to the right.
```

Comment:
I can tell you really had a good time with my `rtl` hack 😂

Reply:
You have no idea how proud I was of myself when it worked - that in itself is a really bad sign 😅

Reply:
TBH I am actually impressed with it, for real! I just ... don't know how it works haha. Want to make sure future me doesn't remove it again :P